### ISSUE TITLE
[Bug] Developer Tools don't work on Linux

### ISSUE THREAD

**Issue by dennisrijsdijk** [crowbartools/Firebot#2007]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

When clicking the Toggle Developer Tools button under Tools, or when using the shortcut Ctrl+Shift+I, nothing happens on Linux. This was tested on both Arch Linux and Ubuntu 22.04

### Expected Behavior

After pressing the UI button or using the keybind, the devtools should open.

### Steps To Reproduce

1. Start Firebot on Linux
2. Click Tools in the top bar
3. Click Toggle Developer Tools
4. Nothing happens

OR
1. Start Firebot on Linux
2. Use the keybind Ctrl+Shift+I
3. Nothing happens

### Version

v5.54.3-linux-x64

### What operating system are you using Firebot on?

Linux Distro

### Relevant log output

```shell
No logs were created.
```

### Contact Details (Optional)

Dennis.sh#9272 or hazrpg#7266

**Comment by zunderscore** [comment]
I'm not able to repro this on my Ubuntu 22.04 VM (i.e. I was able to open the devtools), so I did a little more digging. This appears to be an Electron inconsistency bug (see StackOverflow question [here](https://stackoverflow.com/questions/69969658/electron-devtools-inspector-not-showing-on-linux) and fix [here](https://github.com/electron/electron/pull/31047)). We would need to upgrade Electron to at least 13.5.0 (we're currently on 12.2.2) in order to fix this, and we're not planning on that until v6.

**Comment by hazrpg** [comment]
This affects me too (which is why I've been put down as a contact on this). And as suspected, I think it is the electron version that is the problem - i.e. a bug that's already been fixed in a later version.

Although, part of me is also wonder if its because of the nvidia gpu because I can't run Firebot without first adding `--disable-gpu-sandbox` in the arguments before running. Either in production or development.

**Comment by zunderscore** [comment]
@hazrpg Can you confirm if this works for you in v5.59.0?

**Comment by Wissididom** [comment]
For me, it works. I tested Firebot 5.58.0 where the developer tools didn't open and with Firebot 5.59.0 where the developer tools did open. I tested it with clicking in the menu strip and with Ctrl+Shift+I.

uname -a:
```
Linux dominik-arch 6.6.3-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 29 Nov 2023 00:37:40 +0000 x86_64 GNU/Linux
```

cat /etc/os-release
```
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo
```

So for me, I can confirm.

**Comment by zunderscore** [comment]
I'll go ahead and mark this is as resolved in 5.59.0.
